SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+These versions are currently receiving security updates.
+
+| Version      | Supported          | Notes |
+| ------------ | ------------------ | ----- |
+| 1.0.0        | :white_check_mark: | This is the next major version and has not yet been released. |
```

Comment:
I don't think we plan to do "LTS" branches, so this section may be redundant (I'd keep it simple, and just limit to how to report issues)

Reply:
(we can add a section if we would ever to a `v2`)

Reply:
Agreed.